templates/resources/iam_policy.md.tmpl

```diff
@@ -22,6 +22,10 @@ In the Tanzu Mission Control resource hierarchy, there are three levels and five
 In addition to the direct policy defined for a given object, each object has inherited policies described in the parent objects. For example, a cluster has a direct policy and inherited policies from the cluster group and organization to which it is attached.
 Similarly, a namespace has a direct policy and inherited policies from the workspace with which it is associated.
 
+**Note:**
+The scope parameter is mandatory in the schema and the user needs to add one of the defined scopes to the script
+for the provider to function. At once, the user can define only one scope in one resource.
```

Comment:
Why is for the provider to function clause in the next line, PTAL?

Also please rephrase the second sentence to -

Only one scope per resource is allowed.